src/Sylius/Bundle/CoreBundle/Installer/Provider/DatabaseSetupCommandsProvider.php

```diff
@@ -122,4 +126,13 @@ private function getEntityManager(): EntityManagerInterface
     {
         return $this->doctrineRegistry->getManager();
     }
+
+    private function isSqlite(): bool
+    {
+        $platform = $this->getEntityManager()->getConnection()->getDatabasePlatform();
+        if ($platform instanceof \Doctrine\DBAL\Platforms\SqlitePlatform) {
+            return true;
+        }
+        return false;
```

Comment:
`return $platform instanceof \Doctrine\DBAL\Platforms\SqlitePlatform` ?

Also we could use that namespace instead